### PR TITLE
Feat/avatar badge

### PR DESCRIPTION
**Pull request by bhavyaj12**
Have added 2 components to the main website, landing page was merged previously. Click on docs or get started on landing page to view.

- Have added Avatar and its variants
- Have added Badge and variants
- CSS code has not been refactored yet completely
- Need to add comments, will add them
- Have made somewhat responsive but not fully yet
- Codesandbox link: https://codesandbox.io/s/sleepy-meadow-d1lqx?file=/index.html

### CODE REVIEW

**Review by sakshi006 — COMMENTED** [review]
Looks good to me.

**Review by vikrantkode — COMMENTED** [review]
Looks good to me.

**Comment by sainath7878** [comment]
Side navbar hover color can be better. The active class's text isn't visible while hovering. Rest looks good.

File: css/landing.css [hunk 1]
```diff
@@ -97,27 +22,63 @@ main {
...
+.button-landing a {
+    text-decoration: none;
+}
+
+.btn {
+    text-decoration: none;
+    display: inline-block;
+    border-radius: 0.2rem;
+    border: none;
+    margin-top: 1.5rem;
```
Comment (MaruthiPothuganti) [inline css/landing.css:57]: Suggestion: short hand property can be used

File: css/nav-common.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+}
+
+.subheading-navside a:hover {
+    background-color: var(--cta-color);
+    opacity: 20%;
+    border-radius: 1px;
+}
+
+.component-disp {
+    margin-left: 15rem;
```
Comment (MaruthiPothuganti) [inline css/nav-common.css:123]: Same as above, Can be done by shorthand property

File: index.html [hunk 3]
```diff
@@ -38,24 +44,26 @@
...
+                <div class="heading-main">
+                    <h1>Build stunning projects faster.</h1>
+                </div>
+                <div class="subheading-main">
+                    Halcyon is a CSS component library, with fully functional and modular pre-defined styled classes
+                    that will help you create beautiful UI in a matter of seconds.
+                </div>
+                <div class="button-landing">
+                    <a href="/pages/components/avatar.html">
+                        <button class="btn btn-primary">Get Started</button>
```
Comment (MaruthiPothuganti) [inline index.html:59]: image

File: pages/components/badge.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/nav-common.css">
```
Comment (MaruthiPothuganti) [inline pages/components/badge.html:10]: Suggestion: You can have a common CSS and import all the components into that and you can link that here

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended Changes: Use of short hand property and some change in structuring

File: pages/components/badge.html (hunk 4 above)
Comment (bhavyaj12) [reply pages/components/badge.html:10]: Thanks a lot!! Will work on these changes

**Pull request merged**